content/release-10-18+/applications-10-18+.md

```diff
@@ -264,6 +272,22 @@ Build version</th>
 </tr>
 </thead><tbody>
 
+<tr>
+<td>Cockpit</td>
+<td>In the "Data point graph" widget configuration, you could only select a chart type if the datapoint was not linked to a template from the data point library. Now you can always select a chart type.</td>
+<td>MTM-54007</td>
+<td>10.18.497.13</td>
+<td>UI</td>
+</tr>
+
+<tr>
+<td>Cockpit</td>
+<td>If a user drags the X axis in a "Data point graph" widget with realtime enabled, realtime is turned off. It can be enabled again by clicking the real-time indicator.</td>

```

Comment:
```suggestion
<td>If a user drags the X-axis in a "Data point graph" widget with realtime enabled, realtime is turned off. It can be enabled again by clicking the real-time indicator.</td>

```